README.md

```diff
@@ -26,6 +26,11 @@ to compile the test apps yourself.
 - **iOS**:
   - [Xcode](https://apps.apple.com/app/xcode/id497799835?mt=12) 11.3 or later
   - [CocoaPods](https://cocoapods.org/)
+- **Windows**:
+  - Ensure that Developer Mode is turned on in Windows Settings App
+  - [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+  - [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk/) version 10.0.17763.0 (Windows 10, version 1809) or later
```

Comment:
This should get installed with Visual Studio 2019. I think we can refer users to the RNW docs here: https://microsoft.github.io/react-native-windows/docs/rnw-dependencies